package.json

```diff
@@ -7,6 +7,7 @@
   "type": "module",
   "jsdelivr": "dist/chartjs-adapter-luxon.umd.min.js",
   "unpkg": "dist/chartjs-adapter-luxon.umd.min.js",
+  "main": "dist/chartjs-adapter-luxon.umd.min.js",
```

Comment:
If we're going to add this then it should be the ESM version. Are you on the latest version of the eslint import plugin? It seems like we should file a bug in that repo if so
```suggestion
  "main": "dist/chartjs-adapter-luxon.esm.js",
```